src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -82,9 +83,14 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
             return {it->second};
         }
     } else if (name == ov::supported_properties.name()) {
-        return std::vector<ov::PropertyName>{
-            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
-            ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO}};
+        // return supported_configKeys;
+        std::vector<ov::PropertyName> property_name;
+        property_name.push_back(ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RW});
```

Comment:
I suppose ov::supported_properties &  ov::device::full_name is read only property, only the config keys can be RW.